feature-configs/demo/gatekeeper/sample-testcases.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
How about extending the e2e tests with these?

Reply:
agree that this is needed -- with the 4.9 mixup with gatekeeper I was going to defer that.  If you have time to help walk me through adding the e2e I'd appreciate some help in getting it integrated properly.

Reply:
Sure, let's try to carve some time next week.

Reply:
Let me take a shot at adding this to cnf tests